src/Markdig/Helpers/LegacyHelper.cs

```diff
@@ -0,0 +1,20 @@
+#if NET35
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Markdig.Helpers
+{
+    internal static class LegacyHelper
+    {
+        [MethodImpl(MethodImplOptionPortable.AggressiveInlining)]
+        public static bool HasFlag(this Enum enumeration, Enum flags)

```

Comment:
Couldn't we use instead the new generic constraint for that? (`where TEnum : System.Enum`, likely requiring the project to use C#7.3)

Reply:
I did not know about this addition.
Added the generic constraint now

Reply:
I just added the helper method for net35 since I find the old way of checking for flags unintuitive.